client/src/components/Searchbar.js

```diff
@@ -15,14 +18,38 @@ const Input = styled.input`
   min-width: 284px;
   background-image: url(${SearchIcon});
   background-repeat: no-repeat;
-  background-position: 8px 7px;
+  background-position: 12px 10px;
   font: var(--font-robotolight);
 `;
 
-export default function Searchbar() {
+const ClearButton = styled.div`
+  position: absolute;
+  top: 57px;
+  right: 57px;
+  background: transparent;
+  border: none;
+  &:focus {
+    outline: none;
+  }
+`;
+
+export default function Searchbar({ onSubmit, onClear, ...props }) {
+  const handleClear = (e) => {
+    e.preventDefault();
```

Comment:
Here you `e` and above you use `event`. Try to be consistent with your code 💯

Reply:
Thank you for the reminder